package.json

```diff
@@ -2,10 +2,6 @@
   "name": "@autorest/csharp-v3",
   "private": true,
   "description": "package.json intended for in-repo use only, package.json used for publishing is located in src/AutoRest.CSharp.V3/package.json",
-  "scripts": {
-    "start": "dotnet ./artifacts/bin/AutoRest.CSharp.V3/Debug/netcoreapp3.0/AutoRest.CSharp.V3.dll --server",
-    "debug": "dotnet ./artifacts/bin/AutoRest.CSharp.V3/Debug/netcoreapp3.0/AutoRest.CSharp.V3.dll --server --launch-debugger"
-  },
```

Comment:
If this is the in-repo package.json, why remove the scripts?

Reply:
Because it's not required. The "main" project json is now correct enough to run the server and we avoid 2 entry point.